### ISSUE TITLE
Can't update

### ISSUE THREAD

**Issue by knightian** [Ansuel/tch-nginx-gui#1024]
Trying to 9.6.40 -> 9.6.44

```
Upgrade Script Downloading GUI file...
Upgrade Script ERROR: File corrupted!!!
Upgrade Script ERROR: Removing upgrade file and aborting.
```

**Comment by FrancYescO** [comment]
probably a temporary github problem

**Comment by knightian** [comment]
Yea it is resolved now, am on 9.6.50
